src/lib/helpers.ts

```diff
@@ -24,6 +24,11 @@ import { CursorPageable, getPagingParameters } from "relay-cursor-paging"
 import { formatMarkdownValue } from "schema/v2/fields/markdown"
 import { emptyConnection } from "schema/v2/fields/pagination"
 
+// These default values are only necessary due to caching issues in Gravity.
+// Normally Gravity should always send values for these preferences.
+export const DEFAULT_CURRENCY_PREFERENCE = "USD"
+export const DEFAULT_LENGTH_UNIT_PREFERENCE = "in"
```

Comment:
Moving these constants to this helper file because importing them in `alerts.ts` from `me/index.ts` caused issues.